Update introduction.md (#850)
openhab/openhab-docs: introduction.md

# About openHAB

The  **open H**ome  **A**utomation  **B**us (openHAB,  *pronounced ˈəʊpənˈhæb*) is an open source, technology agnostic home automation platform which runs as the center of your smart home! 

Some of openHAB's strengths are:

- Its ability to integrate a multitude of other devices and systems. openHAB includes other home automation systems, (smart) devices and other technologies into a single solution
- To provide a uniform user interfaces and a common approach to automation rules across the entire system, regardless of the number of manufacturers and sub-systems involved
- Giving you the most flexible tool available to make almost any home automation wish come true; if you can think it, odds are that you can implement it with openHAB.

## What you need to know before you start

**When home automation just seems to work, it is always the result of hard work.** 
Home automation is fascinating and requires a considerable investment of your time. 
Here are some key considerations especially for new users. To be successful, you will need to:

> * **Start slowly and one step at a time.**

Reviewer: I am not sure about the bold usage here.
It looks a bit misplaced and all of those points are highlighted anyway through the list.

Author: Unbolded, no problem.

* Be prepared to learn
* Remain flexible in how you want to achieve your goal
* Celebrate all the small successes

Remember, openHAB is just a computer program. 
The computer will only do what *you* tell him to do.
openHAB can provide many default solutions that are easy to setup. 
On the flip side, the more you insist that everything should look and work exactly as you want it, the more work you will have to do. 
openHAB is fully customizable, but doing so will require substantial efforts from you.

After you have read the documentation for openHAB, you will have:

* Identified on which computer you will run openHAB
* Learned how to install openHAB, as well as all other software that is needed to run openHAB (e.g., JAVA)
* Learned how your smart devices communicate with openHAB; how to make openHAB give commands to your smart devices; and how you can interact with openHAB

Keep your focus. 
For almost everything, there is more than one way in openHAB to achieve a goal or perform a task. 
Initially, this can be frustrating and confusing, but it also gives great flexibility and does not mandate the use of certain devices or tools.

> You may need to learn more about computers that you currently do.

Reviewer: "You may need to learn more about computers **than** you currently do."

Author: Thanks for catching this

But this may not be a problem as you are reading the documentation of software that will enable you to do home automation yourself.

> Lastly, be prepared to start a new hobby: home automation. 

Reviewer: I would add a blank line between here and the next paragraph.

New hobby "home automation" -> *Time to breath* -> Here we go with additional information.

Author: Great, done

Basic functions can be achieved in openHAB rather quickly, e.g., switch lights on at a certain time. 
Others will require much more efforts and thoughts, e.g., how do I determine that someone is home, including guests, but not counting pets? 

> The openHAB forum is a great place to learn and discuss. 

Reviewer: L50 - L53 are a bit redundant compared to the community part..
Was there a background for mentioning this twice?

Author: No real reason, but I remember feeling strong about this issue when I wrote it. deleted

But always remember: no one but you knows what you need and what you have as well as yourself. 
Give plenty of background information if you have a question. 
And please be courteous and respectful, this is an open-source community and everybody on the forum works with openHAB in their spare time.

## A quick overview

openHAB communicates electronically with smart and not-so-smart devices, performs user-defined actions and provides web-pages with user-defined information as well as user-defined tools to interact with all devices. 
To achieve this, openHAB segments and compartmentalizes certain functions and operations. 
The following table gives a high level description of the most important concepts as well as a link to more information:

| Concepts | Meaning | More Information |
| --- | --- | --- |

> | Bindings | are the openHAB component that provides the interface to interact electronically with devices | _see below_ |

Reviewer: Was the limitation to "Bindings" made an purpose?

I am asking because:
In fact Bindings are only a sub category off all add-ons.
Add-ons is the navigation title in paper ui. 
Add-ons is the used term for the website category.

I know that this aims to be a simplified overview, but i would think about using addons, to help people find their way in the website navigation/paper ui (and possibly other places as well).

Author: Very self-centered approach: I remember my confusion about the various add-ons while I was still struggling to understand the basic concepts. Not sure how to include transforms, persistence, integrations, voice and actions without writing a whole para. I wanted to leave it just as a straight line: device connects via binding to thing, which connects to item, which I can see on sitemap. Adding the other add-ons means adding a whole bunch of side chains....happy to get pointers if I am overthinking it. Otherwise, it would be my preference to leave as is

Reviewer: Fine for me, just wanted to talk about it.
We can change it anyway later, when someone has a better approach. :)

## While you are getting started

To all newcomers: please read the section [New User Tutorial]({{base}}/tutorial). 
That section provides you with a step-by-step instructions for your first easy setup. 

OpenHAB runs on most popular platforms such as Linux, Windows and MacOS and on almost any hardware ranging from Raspberry Pis to desktop computers and server PCs. 
You can find specific installation instructions for these and other platforms in the [Installation Overview]({{base}}/installation/index.html) article. 
If you have a strong preference towards a particular platform, then that platform is probably your best choice.

You can install openHAB on your desktop computer for evaluation purposes if you already have any of these systems available for use, but we recommend using a dedicated system in the long run. 
If you feel serious about home automation it may be better to start with a dedicated system right away.

> For anyone undecided: many people find that the simplest way to experiment with openHAB is to get a [Raspberry Pi (Version 2 or 3)](https://raspberrypi.org) and install [openHABian]({{base}}/installation/openhabian.html); a "hassle-free openHAB setup". 

Reviewer: Personally i would not recommend a PI 2 nowadays.
Was there a discussion about it? I didn't follow the complete thread.

Reviewer 2: Jerome:
I asked for a consensus in the user forum thread discussing this issue. @rkoshak has replied, await a few comments from the rest of the regulars but the proposed text seems to strike a good balance. I have zero experience with Pi so feel unable to comment on it's suitability as 'the' recommended platform and I was never sure if a recommendation should be made at all. I defer to those more experienced then me.

Author: Agree with @Confectrician ....I understand that others may disagree, but then the intent here is to give a recommendations for those who do not yet know otherwise. Will change to "(ver 3 for best experience)". I have a few ver2 as mediaplayers....and goodness are they slow....so if a user has no other preferences, I have no idea why we would subject her/him to an experience with a ver2....power users will contest this....but then this section is not written for them, they know better anyway.....and will not be mislead by whatever this section states.

> For anyone undecided: many people find that the simplest way to experiment with openHAB is to get a [Raspberry Pi (Version 2  or 3)](https://raspberrypi.org) and install [openHABian]({{base}}/installation/openhabian.html); a "hassle-free openHAB setup". 

Reviewer 3: >openHABian - the hassle-free... 

?

Reviewer 2: The term 'hassle free' has been drawing some fire on the user forum in threads discussing this topic. See thread
(https://community.openhab.org/t/new-docs-discussion-should-we-recommend-a-platform/62822/73)
In fact it seems you are credited as the author - see post 73

> While openHABian offers a streamlined and simplified way to get up and running quickly, it is a complete openHAB home automation system easily capable of automating your entire home.

Reviewer 3: I'm not sure what you want to say with "While... it is a complete openHAB home automation system". Instead maybe mention that it is based on a normal Linux distribution thus helping users from the beginning to the full-blown setup

Author: ```suggestion
openHABian offers a streamlined and simplified way to get up and running quickly and provides a complete openHAB home automation system easily capable of automating your entire home.
```

Author: Fixing sentence, hope that is acceptable for both of you

> However, it is worth noting two main limitations for Raspberries. 

Reviewer 3: ```suggestion
However, it is worth noting two potential limitations of Raspberry Pis. 
```

I read the sentence wrong at first. This might be clearer.

Reviewer 2: 'main' limitation sounds to absolute, potential is much better

Author: works for me

> They are limited in the RAM (memory) and may not perform well when additional database and data visaulization programs are installed.

Reviewer 3: Remove "the" in front of RAM.

Author: ```suggestion
They are limited in RAM (memory) and may not perform well when additional database and data visaulization programs are installed.
```

Author: Fixed, thanks

> Running Raspberries from SD-cards only may result in system instabilities as these memory cards can degrade quickly under openHAB's use conditions. 

Reviewer 3: This is not true. Yes it is wise to think about backups and yes there have been users with bad luck. This is not the general case, otherwise someone needs to explain to me why the 11th iteration of the Raspberry Pi is still entrusted with an SD card 😉

Reviewer 4: > This is not true. Yes it is wise to think about backups and yes there have been users with bad luck. This is not the general case, otherwise someone needs to explain to me why the 11th iteration of the Raspberry Pi is still entrusted with an SD card 😉

Because Pis are build to learn programming. They are not meant to be 24x7 servers.
I know you don't believe in it and yes it is not a compelling result and it depends on a number of factors if it happens (too difficult to discuss in that here or in any introductory article), but for sure there's way too many people to have suffered from this than for this to be "not true" either. Also note that "use conditions" also include power outages which are frequent in HA and another way to crash SD cards.
I think this general warning covers both and should therefore remain in the docs as it stands now.
Omitting it would frustrate a number of people when their system crashes and they would attribute that to openHAB, not their Pi.
And yes, the install instruction should be reworked with a more extensive explanation, too.

Author: I understand that the discussion can be more nuanced and complicated. This part is the very first introduction, we simply cannot overload it with too many technical details. This intro is also deliberately written to connect with real novices. Anyone with more expertise will just glance at it sigh and move on. I am agreeing with @mstormi and suggest to keep as is
